src/com/spikes2212/genericsubsystems/drivetrains/commands/DriveTankWithPID.java

```diff
@@ -11,106 +11,81 @@
 import edu.wpi.first.wpilibj.command.Command;
 
 /**
- * This command moves a {@link TankDrivetrain} using wpilib's
- * {@link PIDController}. It also waits a specified amount of time after the
- * error is within the given tolerance before stopping the PID Loop to make sure
- * the drivetrain doesn't go past and remain beyond the setpoint.
+ * This command moves a {@link TankDrivetrain} using wpilib's <a href=
+ * "http://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/PIDController.html">PIDController</a>.
+ * It also waits a specified amount of time after the {@link TankDrivetrain} is
+ * within the given tolerance before stopping, to make sure the
+ * {@link TankDrivetrain} doesn't go past the setpoint.
+ *
```

Comment:
the last doc was better because it used more accurate words (such as "error"). bring it back and just fix the links